frontend/packages/shipwright-plugin/src/components/build-list/BuildTable.tsx

```diff
@@ -163,16 +166,27 @@ export const BuildTable: React.FC<BuildTableProps> = (props) => {
     }),
     [buildRuns, buildRunsLoaded, buildRunsLoadError],
   );
+  const buildResource = props.data.map((sBuild) => {
```

Comment:
I like the idea of changing the data instead of using customData. But can we do this as part of the `useMemo`? And update the naming from `customData` to `data` maybe if we don't use `customData` anymore. Let me know if you want chat on this.